docs/notebooks/diagnostics/charged_particle_radiography_particle_tracing.ipynb

```diff
@@ -0,0 +1,552 @@
rqap599lm6urp4+umnmZubw+v1cuWVV9Le3k5xcfG+DMIZZKQ9Afr6+rIjQ/sZGSUuIBuQc5kRZ2CxWDh48CCLi4uMjo6etZbmeu73N8kCEX6NM+IMRFHks5/9LG9+85v527/9W/7qr/5qwwCa6VtlfEjz8/MpKCjY0L2ltbWVvr6+rIj6TtBQ5OBYMMrSao/UazfRUOTAKEkIukzyjLIyQDAs41rNcjO90eVYmkXptZu475oDZ50nA4dZ4lC5i+dOreCwSMRSKs0VBSxOjJJSqhkLRjFKIoIAiqqj6ekM1SAKVHltvGbV5/e25gKuqc5nIZri+wPzPNQzy0IkhdEg8tffHWZ6OUkoISNrOrKiIZCe+dXTxWxMksDV1flEVlW+EopGns2IKaViNaavy7jqZ2w1ihydj7ISiRFeXsx+N16vl5KSkqx95dTUFP39/ZuWRvc7DAYDnZ2d9Pf3k0qltsVUvdg4sx3g9XpZWlqiqanpki2319XVMTk5mVUT2+9WfGtL1Yqi5FRYYy0yMq4nT56kr6+P5ubm7MYq18E/HA5fklacO8Wlt2rtAJIk8fnPf57Xv/71fPSjH+V973sfqqoyPj6OpmksLCxgMBjw+XzU19efl8maIar09fUB7CgYH67I52QwRkLRGDJFENApzbOyEEnhthlxms/+ag54rTxzaplSo4iqpzWbC91bG4ERBIHfv7KcYreF44EYxW4zN9R5SSYTfO+XvVgMAooOkWR69tciQrXfTiSp8vqrymkqcqJqOv/z9ERWzaos34IoCgiiQErVePrkMoqqY5TAKEkkFA0N8FgN2EwGErLKNTX56MD/PD2JgM5STEbVdGxGiWuq8/n58SX01Sw8KSug64wOD1JQULApCa60tBSDwUBXVxd55fU82D1HStF4WUcxB8vPLvHvR6yVxLxQ5iM7wUb9Xr/fv64dkEgk6O3tvaTL7WVlZVit1uxs7H5nt2dIXKFQiHg8vmfPjiAIVFVVMT8/T3d3Ny0tLdhstpyXw3/TStO/duNL50IwGOQlL3kJuq4zNzfHPffcw1vf+la8Xu+OMqnMbGuG9bldzIWSDM6EiKZUgpEUcVnDbBBoLHRQ6LJQ6bWtE7ZIyCrf6Z1ldD6CrsON9T5eULszluePhuf5v545dF1H1HVOBRZx2OwcCyYQRbCZDFxRmcfsSoJXHS7hztYiHuqd4ctPT+J3pOeWe6ZC6ICiahglkZSiEZc1RCGt+ZyRhpYEsJsNFDrNfOIVzTw6NM+3u2cxSgITSwnS1LI0Ch0i8VTaU1qURP7y9jpub92aMEHXsSne/MBRVv0sMBtEPvWq1h3fo4uBtWMj22Ei7yU26/eea75XURR6e3vx+Xw7rhrtB0QiEfr7+/ctoU7XdUKhUHZjZDabs3oFufA2PhfC4TBDQ0PU1tZitVo5duxYtrS/W3zoQx/i5ptv5rbbckZo3n+72jX4tc+I4/E4n/nMZ/j+979PMpnk5ptv5tlnn+VlL3sZ73znO3e1cxRFkfb2dnp6ehAEYduWd4UuM4Wu07N+vVMr/GgowEO9s0RSKi1FTu67tgKzIf0yWYwSh8vdnFyIkVI0jgWiNBQ66J0OMb4Qx+80c2O99yxPY0hbL04sxXFaDNhMEt88MkuBy4RREpleTmAw2UjGY+RZjYQSKgfyLSiqhg6U56ez0O7JEHaTYXVzIGA1SizFZKTVWKGzatwgpoOwJKQZ4h67kZSq855bakmpGt/smkHVIZZSsjs9QUgfIBjV+My9rSwnVJqKHNQVbH1X/J2RMEk1XQpPW0BqfPqnY5dUIM4Q0cbHx+nu7qajo+OiKD9tNt9bV1e3pc1Bptw+ODjI0aNH98xEZa/hcDg4ePAgPT09lJWVXRCi0vmQ2RjNz8+ztLSUnRleO5qXGW9aS+LKNTLM//7+/l1Z0W6EvWJN71f82gdik8mE3+/n/vvvz0q+JZNJXvnKV2KxWHjLW96y62Dc0dFBT08PwI79Z1VN58nRBebCqTT72SDyy7ElfE4TrzqUHqdYjsl8s3sGt8WI1SQxH07y9z8cxbFqcTi+FGdyOc5911Rgkk6/FE+NLfK156YRhfR57CYJXUgzn4G0xzA611QVMjI2zjJOwjIsxmReebCEtpI0acJnNzE0EyFDocizGRAEWIjKqJqGuBpWZS1NDpNE8NhNlOZZCCcURufDfPXZKRajKVRtfbklEzx1QeAFdb4NR5fOd/+eOLqQJaWpenoLnJS3byO3H1BRUYHRaLygfcqN+r27me8VRZGWlhaOHTtGX18fra2t+yLD3y7MZjOHDx+mr6+PeDxOdXX1Bd9UqKrKwsIC8/Pz2Y3RZhwWIDtfnEqlkGV5x97G54PJZKKzszN7b3LVK/5NK03/2gdiSZJ4wxvesO7vzGYz3/jGN7j77rsxGo288Y1vzGlmvJM5RFXXiaZUwgmZfHt60XWaJUbno8TlNJEpGE2haWA1rYp0WI38/NgCt7cUIIkiTouB6eUEgXCS0rx0vy6clLn/2Sm8NhM2swS6ztBcBFnVVw0kBCaW4kyvJNEACTfXeiJc3VZHkd+7TvDjVYdL6JsOE4ykCWJleTY+c28tjw8HePL4IsOzEfxGEYMkMLmUQNV1nGaJlWgCXVP55rNj6JpIkdPEbEReZYunoZMOnnU+67aDMMATowusJNZ75urAXc1b05zej8iwjDMzurmWxNxKv3e3yJCfJiYm6Orqor29fV8zpzeDJEl0dHRw9OjRC+aklUqlssIniURiR+5fZ5pG7MU1i6JIWVlZ1tikpaVl1xvH37Kmf0NgsVj41re+xUtf+lKMRiOvfe1rd3W8zIuaCcbbLWGZJJFKr43+6TBOi0ZC1rAYpfSY0OrP2E0SGqcDaDylYJBEMu0PXU9zkzOBbHAmzJefnqBrYoWUrOK2mSj3WHFZDFTkWzkejKHrcHIhTmORA5/DTCyl8vNFByUnx7AbBSxrZPL8jnSPd2A6jChAS4
```

Comment:
```suggestion
    "[create_particles()]: ../../api/plasmapy.diagnostics.charged_particle_radiography.Tracker.rst#plasmapy.diagnostics.charged_particle_radiography.Tracker.create_particles\n",
```

Reply:
Not sure why GitHub is displaying the wrong line here, but if you go to the "Files" tab you can see this suggestion in the proper context.